app/assets/stylesheets/all/components/_forms.scss

```diff
@@ -19,6 +19,19 @@ form.form_form {
   }
 }
 
+// Override bootstrap on dl's and dt's for read only questioning
+dt {
+  margin-top: 20px;
+}
+
+dt:first-of-type {
```

Comment:
These style should be on `dl` (the outer tag), not `dt`. I'm not sure how this is working currently.

Reply:
Let's talk about this Fri am. <dl> wraps around all the skip rules, and the idea is to put space after each skip rule. Each skip rule is in a <dt>